src/RcppExports.cpp

```diff
@@ -6,6 +6,11 @@
 
 using namespace Rcpp;
 
+#ifdef RCPP_USE_GLOBAL_ROSTREAM
+Rcpp::Rostream<true>&  Rcpp::Rcout = Rcpp::Rcpp_cout_get();
+Rcpp::Rostream<false>& Rcpp::Rcerr = Rcpp::Rcpp_cerr_get();
+#endif
```

Comment:
What is the purpose of these lines? Are they related to the curl issue, or is it something else?

Reply:
These lines were added automatically by devtools when I ran `devtools::install()`. Honestly I don't know what the purpose of these lines is and can undo it if you want me to.

Reply:
Ah, yes, it's an autogenerated file, so probably just some update. No worries, just leave it as is.